lib/Grape.js

```diff
@@ -22,7 +20,7 @@ class Grape extends Events {
       dht_maxTables: 5000,
       dht_maxValues: 5000,
       dht_port: 20001,
-      dht_bootstrap: [],
+      dht_bootstrap: undefined, // defaults to hyperswarm public bootstrap servers
```

Comment:
do we want the default to use public bootstrap servers?

Reply:
usually we run it in private networks

Reply:
for sure - but as an OSS library it could be that public was okay - I'll change to private

Reply:
```suggestion
      dht_bootstrap: [],
```